test/spec/AlignToOrigin.spec.js

```diff
@@ -190,4 +191,37 @@ describe('alignToOrigin', function() {
     expect(elementRegistry.get('Event')).to.exist;
   });
 
+
+  it('should allow alignment with auto-save', async function() {
```

Comment:
What's being tested here?

Reply:
It is just a visual test for now, but I guess that one could be improved.

The idea is that you can model without the editor blowing up and still get automatic saving. 

Will improve the test case.

Reply:
I've added https://github.com/bpmn-io/align-to-origin/pull/5/commits/f712a2adc7e4bfea01fa574afdef4391d2f97776 but I'm also happy to remove that spec all together.

What I've found during testing is that the looping is hard to reproduce in a test case anyway, as it requires user input/delays to trigger the actual error.

Reply:
No worries, I was just trying to understand your thought process. 😄